src/swa.d.ts

```diff
@@ -71,6 +71,9 @@ declare interface SWACLIEnv extends StaticSiteClientEnv {
   AZURE_TENANT_ID?: string;
   AZURE_CLIENT_ID?: string;
   AZURE_CLIENT_SECRET?: string;
+
+  // swa telemetry
+  SWA_DISABLE_TELEMETRY?: string;
```

Comment:
SWA_DISABLE_TELEMETRY -> `SWA_CAPTURE_TELEMETRY` ?

Reply:
As this is something users can set/reset to disable the telemetry capturing, i think SWA_DISABLE_TELEMETRY makes more sense no?

Reply:
ok.. then lets scope the string value to two values then ? ( true/false )

Reply:
users can change the env variable value right, how can we control that? added case insensitive comparision in the code.

Reply:
> SWA_DISABLE_TELEMETRY -> `SWA_CAPTURE_TELEMETRY` ? in this case.. they will set it to false.. default value of it will be true

CAPTURE_TELEMETRY makes sense. I agree with Sulabh.

Reply:
bu all the other CLIs also have the env variable name as DISABLE_TELEMETRY

Reply:
The reason why I gave the suggestion of using SWA_CAPTURE_TELEMETRY was every where in the docs/command help text we are talking about... Capture telemetry ..  it was more natural to have command in that format.

https://learn.microsoft.com/en-us/cli/azure/azure-cli-configuration

Reply:
Also noticed  we are using .. `SWA_CLI..` as the prefix for the env variables. if we are deciding on any name lets include `SWA_CLI` as the prefix for sure